SimPEG/potential_fields/magnetics/simulation.py

```diff
@@ -254,37 +254,39 @@ def evaluate_integral(self, receiver_location, components):
         Tz = [Tzx Tzy Tzz]
         """
         # TODO: This should probably be converted to C
-        tol1 = 1e-10  # Tolerance 1 for numerical stability over nodes and edges
-        tol2 = 1e-4  # Tolerance 2 for numerical stability over nodes and edges
+        tol1 = 1e-4  # Tolerance 1 for numerical stability over nodes and edges
+        tol2 = 1e-10  # Tolerance 2 for numerical stability over nodes and edges
```

Comment:
@dccowan We'd love your input on what these two `tol` parameters really mean, and why there are two with different values? I have this update, and @domfournier has another (https://github.com/nwilliams-kobold/simpeg/pull/1) that simplifies them to just one? Are we on the right track?

Reply:
@nwilliams-kobold I suggest merging https://github.com/nwilliams-kobold/simpeg/pull/1, and let unitest run. We can modify it back here if peeps have issue with it.

Reply:
Cool, all systems green on this end. We will give a chance to @dccowan to chime in, but it looks fine by me.